docs/content/3.application-side/2.session-access-and-management.md

```diff
@@ -103,12 +105,16 @@ await signIn(credentials, { callbackUrl: '/protected' })
 // Trigger a sign-in with a redirect afterwards to an external page (if set, this will cause a hard refresh of the page)
 await signIn(credentials, { callbackUrl: 'https://sidebase.io', external: true })
 
+// Trigger a sign-in without calling getSession directly. You have to manually call it to get session data.
+await signIn(credentials, { withGetSession: false })
```

Comment:
Can we rename this new option from `withGetSession` to just `getSession`? I think this would be a bit more readable!

Reply:
Hi Zoey, you just caught me on holiday 🌴. I will change it asap, next week for certain. I will try to do it sooner. 

Renaming is fine by me. I don't totally agree with the name change. Because it is a boolean you want to name it like `is...`, `has...`, `with...` or something like that, and not a get/set/action like method name. 

So I would suggest just `withSession` or without 'with', e.g. something like `callSession` or just `session`. But again all names are fine by me. Just giving my opinion to get it as good as possible.

Reply:
Agreed! Let's go with `withSession`

Reply:
Done in [`ac9415f` (#702)](https://github.com/sidebase/nuxt-auth/pull/702/commits/ac9415fb6ff552478872c391badbdd1d5bf11d86)